writegood-mode.el

```diff
@@ -152,6 +152,12 @@
   :group 'writegood
   :type '(repeat character))
 
+(defcustom writegood-on-the-fly-modes
```

Comment:
I'm no sure I understand why we would want to restrict the modes here?

If I turn on writegood in any mode and then toggle on the on-the-fly feature, it should work. Is there a reason to add this filtering/checking step?

Reply:
So far this mode acts globally when you toggle it. This means that if you are programming, writegood-mode would try to calculate the score of your code (which does not seem to make sense). For this reason I restricted to only those modes I found this feature useful.

A sensible alternative would be to run the feature only for the current buffer.

I just like it activated by default in multiple modes because I do not need to think about activating useful stuff while I am focusing on writing. Sometimes I just think: "this sentence does not look good" and then I find confirmation in the sentence score in the modeline :)